interface/modules/zend_modules/module/Installer/src/Installer/Model/InstModuleTable.php

```diff
@@ -73,16 +73,18 @@ public function getConfigSettings($id)
    */
     public function installSQL($dir)
     {
-        $sqltext = $dir . "/table.sql";
+        $sqltext = (file_exists($dir . "/table.sql") ? $dir . "/table.sql" : $dir . "/install.sql");
```

Comment:
ok, this answers my question above regarding table.sql since it appears this is from previous mechanism

Reply:
if that is case, though, shouldn't above go to $dir . "/sql/install.sql" when no table.sql ?

Reply:
The line below checks for the presence of the file.